recipes/segway/meta.yaml

```diff
@@ -41,6 +39,7 @@ test:
 
   commands:
   # Smoke test
+  - segway --version
```

Comment:
Do we need to grep for the version string in the output? I assume this was giving something wrong previously, yes?

Reply:
You are correct I probably should check against `{{ version }}`. I will likely update shortly though for now, I would like to manually verify the output is correct then check to see if the grep test works.

And yes you correct, in the current conda environments the version string reported is `0.0.0`. Otherwise the version string is correct (when installed through pip, setup.py, etc).